iterations.r

```diff
@@ -46,24 +46,25 @@ source("functions/data_tests.r")
 test_stations_metadata(stations_metadata_df)
 
 
-### 5: Final volume query: 
+### 5: Final volume query:
 
 source("gql-queries/vol_qry.r")
 
-stations_metadata_df %>% 
-  filter(latestData > Sys.Date() - days(7)) %>% 
-  sample_n(1) %$% 
+stations <- stations_metadata_df %>%
+  filter(latestData > Sys.Date() - days(7)) %>%
+  sample_n(1)
+stations %$%
   vol_qry(
     id = id,
     from = to_iso8601(latestData, -4),
     to = to_iso8601(latestData, 0)
-  ) %>% 
+  ) %>%
   GQL(., .url = configs$vegvesen_url) %>%
-  transform_volumes() %>% 
-  ggplot(aes(x=from, y=volume)) + 
-  geom_line() + 
+  transform_volumes() %>%
+  ggplot(aes(x = from, y = volume, color = stations$name)) +
+  geom_line() +
+  ggtitle("Traffic volume over timeperiod") +
+  xlab("Timeperiod") +
+  ylab("Volume") +
+  labs(color = "Station") +
   theme_classic()
```

Comment:
Great plot! 

Consider adding the name of the plot in the title. You can e.g use the "paste" function for this. Since we only have one line, that would be more convenient than having a legend. Usually you only use a legend if there are multiple categories/lines in the chart.